File: src/main/resources/static/itemTransfer/itemTransferService.js
var services = angular.module('itemTransfer.services', [ 'ngResource' ]);

services.service('itemTransferService', [ '$http', function($http) {

	this.findAll = function() {
		return $http.get("/itemTransfer");
	}
	
	this.save = function(itemTransfer) {
		return $http.post("/itemTransfer",itemTransfer);
	}
	
	this.next = function(id) {
		return $http.get("/nextPlaces/" + id);
	}
	
	this.delete = function(itemTransfer){
		return $http.get('/itemTransfer/deleteItemTransfer/' + itemTransfer)
	}

	this.findAllAnalyticsOfStatements = function(){
		return $http.get('/analyticsOfStatements');
	}
	
	this.findAllInterbankTransfers  = function(){
		return $http.get('/interbankTransfer');
	}
	
	this.search = function(item){
		return $http.post('/itemTransfer/search',item);
	}
}]);

File: src/main/java/bank/itemTransfer/ItemTransferService.java
package bank.itemTransfer;

import java.util.List;

public interface ItemTransferService {
	List<ItemTransfer> findAll();

	ItemTransfer save(ItemTransfer ItemTransfer);

	ItemTransfer findOne(Long id);
	
	public void delete(Long id);
	
	public List<ItemTransfer> search(ItemTransfer itemTransfer);
}
